src/Datadog.Trace/Agent/IApi.cs

```diff
@@ -1,9 +1,12 @@
+using System;
 using System.Threading.Tasks;
 
 namespace Datadog.Trace.Agent
 {
     internal interface IApi
     {
+        public void OverrideBaseEndpoint(Uri baseEndpoint);
```

Comment:
Consider renaming to `SetBaseEndpoint()`.

Reply:
Good idea, implemented in a8ae6f5ec783d182198779a59a280a26eb12a5d1